README.md

```diff
@@ -0,0 +1,39 @@
+# UERANSIM snap
+
+This project is contains a snap package of UERANSIM.
+
+UERANSIM (pronounced "ju-i ræn sɪm"), is the open source state-of-the-art
+5G UE and RAN (gNodeB) simulator. UE and RAN can be considered as a 5G
+mobile phone and a base station in basic terms. The project can be used
+for testing 5G Core Network and studying 5G System.
+
+## Usage
+
+Install and connect the snap:
+
+```bash
+sudo snap install ueransim
+sudo snap connect ueransim:network-control
+```
+
+To run the gNodeB, create a configuration file in
+`/var/snap/ueransim/common/gnb.yml` and run the command:
```

Comment:
Should we have a file there already with a reasonable default?

Reply:
Do you know a way of installing files in there? It is technically outside of the snap so users can write to it and the confined snap can also read it.

I can easily add example config files and change the instructions for users to copy them and edit them instead. They contain IPs anyway, so they will need to be edited in most cases.

Reply:
Yes, if you look at the Vault snap for example we place a basic vault.hcl in this directory so that users can right away run `snap start vault`. 

https://github.com/canonical/snap-vault/blob/master/service/vault.hcl

Reply:
Thanks, I added default config files and updated the README.